### PR TITLE
Various fixes: add sanity check to GEOS and change homepage netcdf4-python

### PR DESCRIPTION
**Pull request by wpoely86**
As requested in #2221

### CODE REVIEW

File: easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goalf-1.1.0-no-OFED.eb [hunk 1]
```diff
@@ -6,9 +6,14 @@ version = '3.3.5'
...
 
-source_urls = [ 'http://download.osgeo.org/geos/' ]
+source_urls = ['http://download.osgeo.org/geos/']
 sources = [SOURCELOWER_TAR_BZ2]
 
 toolchain = {'name': 'goalf', 'version': '1.1.0-no-OFED'}
 
+sanity_check_paths = {
+    'files': ['bin/geos-config', 'lib/libgeos.so', 'lib/libgeos.a', 'include/geos.h'],
+    'dirs': ["."]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goalf-1.1.0-no-OFED.eb]: change this to empty list please, `.` will always be a non-empty dir

**Comment by boegel** [comment]
and make the title a bit more descriptive, otherwise it'll be a nuisance when composing the release notes

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 10 out of 12 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d1075f8acecbbece34ac for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goalf-1.1.0-no-OFED.eb [hunk 2]
```diff
@@ -4,11 +4,16 @@ name = 'GEOS'
 version = '3.3.5'
 
 homepage = 'http://trac.osgeo.org/geos'
-description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goalf-1.1.0-no-OFED.eb]: missing `"""` in front, syntax error

File: easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goolf-1.4.10.eb [hunk 3]
```diff
@@ -4,10 +4,16 @@ name = 'GEOS'
 version = '3.3.5'
 
 homepage = 'http://trac.osgeo.org/geos'
-description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+description = GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-goolf-1.4.10.eb]: same here

File: easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-ictce-4.0.6.eb [hunk 4]
```diff
@@ -4,11 +4,16 @@ name = 'GEOS'
 version = '3.3.5'
 
 homepage = 'http://trac.osgeo.org/geos'
-description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+description = GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-ictce-4.0.6.eb]: same here

File: easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-ictce-5.3.0.eb [hunk 5]
```diff
@@ -4,12 +4,16 @@ name = 'GEOS'
 version = '3.3.5'
 
 homepage = 'http://trac.osgeo.org/geos'
-description = """GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
+description = GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)"""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.3.5-ictce-5.3.0.eb]: same here

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 24 out of 59 (5 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9168bc7d87226ed264c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2c017b6abfd7eadb5aa5 for a full test report.

**Comment by boegel** [comment]
going in, thanks @wpoely86!

**Pull request merged**